website/docs/r/storage_account.html.markdown

```diff
@@ -183,16 +183,26 @@ A `blob_properties` block supports the following:
 
 * `restore_policy` - (Optional) A `restore_policy` block as defined below. This must be used together with `delete_retention_policy` set, `versioning_enabled` and `change_feed_enabled` set to `true`.
 
+-> **NOTE:** `Storage` kind Storage Account doesn't support `restore_policy`.
+
 * `versioning_enabled` - (Optional) Is versioning enabled? Default to `false`.
 
+-> **NOTE:** `Storage` kind Storage Account doesn't support `versioning_enabled`.
+
 * `change_feed_enabled` - (Optional) Is the blob service properties for change feed events enabled? Default to `false`.
 
+-> **NOTE:** `Storage` kind Storage Account doesn't support `change_feed_enabled`.
+
 * `change_feed_retention_in_days` - (Optional) The duration of change feed events retention in days. The possible values are between 1 and 146000 days (400 years). Setting this to null (or omit this in the configuration file) indicates an infinite retention of the change feed.
 
+-> **NOTE:** `Storage` kind Storage Account doesn't support `change_feed_retention_in_days`.
+
 * `default_service_version` - (Optional) The API Version which should be used by default for requests to the Data Plane API if an incoming request doesn't specify an API Version.
 
 * `last_access_time_enabled` - (Optional) Is the last access time based tracking enabled? Default to `false`.
 
+-> **NOTE:** `Storage` kind Storage Account doesn't support `last_access_time_enabled`.
```

Comment:
can we update these to be:

```suggestion
-> **NOTE:** This field cannot be configured when `kind` is set to `Storage` (V1).
```

Whilst I appreciate the Service Team aren't using the V1 label, I think omitting it here would cause confusion that this isn't supported in V2 too, so I think it's worth calling that out?